SpringParking-parkingBranch/src/main/java/pl/springDemo/demo/control/OperatorController.java

```diff
@@ -0,0 +1,23 @@
+package pl.springDemo.demo.control;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import pl.springDemo.demo.userPayment.UserPaymentService;
+
+@RestController
+@RequestMapping("/operator")
+public class OperatorController {
+
+    private final UserPaymentService userPaymentService;
+
+    @Autowired
+    public OperatorController(UserPaymentService userPaymentService){
+        this.userPaymentService = userPaymentService;
+    }
+
+    @GetMapping("/{id}")
+    public Boolean checkUserTicket(@PathVariable("id") String id){
+        long userId = Long.parseLong(id);
```

Comment:
czemu long nie jest w deklaracji metody?

Reply:
Odnośnie wszystkich longów pierwotnie jako parametr wszystkie id były deklarowane jako long, potem poprawiając kod chcąc usystematyzować pobieranie wszystkich parametrów zmieniłam to na String, który następnie mogę parse'ować. Chcąc ulepszyć kod trochę go tutaj przekombinowałam co odejmuje kosmetyce i dodaje niepotrzebne linie kodu.